ocaml/xapi/console.ml

```diff
@@ -129,7 +127,7 @@ let ws_proxy __context req protocol address s =
               let wsprotocol = match ty with
                 | Ws_helpers.Hixie76 -> "hixie76"
                 | Ws_helpers.Hybi10 -> "hybi10" in
-              let message = Printf.sprintf "%s:%s:%d" wsprotocol protocol port in
+              let message = Printf.sprintf "%s:%s:%s" wsprotocol protocol port  in
```

Comment:
Now that `port` can be a port or a path, it may be better to give it another name.